src/components/Layout/Layout.scss

```diff
@@ -669,13 +669,6 @@ header {
 .blog-image {
     &:before {
         background: linear-gradient(rgba(0, 0, 0, 0.75) 0%, rgba(0, 0, 0, 0.25) 100%);
```

Comment:
Thoughts on moving this to Tailwind too? Might be cool to just have a reusable gradient cntl.

Reply:
Absolutely, I just couldn't figure it out in 2 mins on my own. 😇